### ISSUE TITLE
Parenthesis added when entering variables even with feature disabled.

### ISSUE THREAD

**Issue by elkinm** [spyder-ide/spyder#20554]
### What steps will reproduce the problem?

Make variable in code or in a loop.
Use said variable in any way such as print(COUNT()). The () are added after the variable most (but not all the time)
The code fails to run since it is a variable, not a method.

I have enter parenthesis dissabled in the editor but it still happens.

A variable should never have () added, a function or method can and should have () added.

## Versions

* Spyder version: 5.3.3  (conda)
* Python version: 3.9.13 64-bit
* Qt version: 5.15.2
* PyQt5 version: 5.15.7
* Operating System: Windows 10

### Dependencies

```
# Mandatory:
atomicwrites >=1.2.0          :  1.4.0 (OK)
chardet >=2.0.0               :  4.0.0 (OK)
cloudpickle >=0.5.0           :  2.0.0 (OK)
cookiecutter >=1.6.0          :  1.7.3 (OK)
diff_match_patch >=20181111   :  20200713 (OK)
intervaltree >=3.0.2          :  3.1.0 (OK)
IPython >=7.31.1;<8.0.0       :  7.31.1 (OK)
jedi >=0.17.2;<0.19.0         :  0.18.1 (OK)
jellyfish >=0.7               :  0.9.0 (OK)
jsonschema >=3.2.0            :  4.16.0 (OK)
keyring >=17.0.0              :  23.4.0 (OK)
nbconvert >=4.0               :  6.4.4 (OK)
numpydoc >=0.6.0              :  1.5.0 (OK)
paramiko >=2.4.0              :  2.8.1 (OK)
parso >=0.7.0;<0.9.0          :  0.8.3 (OK)
pexpect >=4.4.0               :  4.8.0 (OK)
pickleshare >=0.4             :  0.7.5 (OK)
psutil >=5.3                  :  5.9.0 (OK)
pygments >=2.0                :  2.11.2 (OK)
pylint >=2.5.0;<3.0           :  2.14.5 (OK)
pyls_spyder >=0.4.0           :  0.4.0 (OK)
pylsp >=1.5.0;<1.6.0          :  1.5.0 (OK)
pylsp_black >=1.2.0           :  1.2.1 (OK)
qdarkstyle >=3.0.2;<3.1.0     :  3.0.2 (OK)
qstylizer >=0.1.10            :  0.2.2 (OK)
qtawesome >=1.0.2             :  1.2.2 (OK)
qtconsole >=5.3.2;<5.4.0      :  5.3.2 (OK)
qtpy >=2.1.0                  :  2.2.0 (OK)
rtree >=0.9.7                 :  1.0.1 (OK)
setuptools >=49.6.0           :  65.6.3 (OK)
sphinx >=0.6.6                :  5.0.2 (OK)
spyder_kernels >=2.3.3;<2.4.0 :  2.3.3 (OK)
textdistance >=4.2.0          :  4.2.1 (OK)
three_merge >=0.1.1           :  0.1.1 (OK)
watchdog >=0.10.3             :  2.1.6 (OK)
zmq >=22.1.0                  :  23.2.0 (OK)

# Optional:
cython >=0.21                 :  0.29.32 (OK)
matplotlib >=3.0.0            :  3.6.2 (OK)
numpy >=1.7                   :  1.23.5 (OK)
pandas >=1.1.1                :  1.4.4 (OK)
scipy >=0.17.0                :  1.10.0 (OK)
sympy >=0.7.3                 :  1.11.1 (OK)
```

**Comment by dalthviz** [comment]
Hi @elkinm thank you for the feedback! Could it be possible for you to submit a GIF or video to better understand how this parenthesis addition happens? 

Any other info in order to debug this is greatly appreciated!

**Comment by elkinm** [comment]
![Spyder parentheses](https://user-images.githubusercontent.com/5421747/220470164-df2ce0d0-9571-448e-8686-e3fc1b89a8b4.gif)
Here is a GIF of my issue. I noticed that if COUNT is not used it does not have (), but if COUNT is incremented in the loop or otherwise used in the code then it results in print(COUNT()). I just home the GIF works. Thank you.

**Comment by dalthviz** [comment]
Thank you for the GIF @elkinm ! I was unable to reproduce this with the latest release (Spyder 5.4.2). Could it be possible for you to update Spyder and try again?

To update/use a more recent Spyder version you will need to create a new conda environment and install there Spyder from the conda-forge channel. Let us know if updating helps!

**Comment by elkinm** [comment]
I was unable to install 5.4.2 so far. I followed the environment instructions as outlined in: [https://docs.spyder-ide.org/current/installation.html](url) and then ran: conda install spyder=5.4.2  and recieved:
Collecting package metadata (current_repodata.json): failed

CondaSSLError: OpenSSL appears to be unavailable on this machine. OpenSSL is required to
download and install packages.

Exception: HTTPSConnectionPool(host='repo.anaconda.com', port=443): Max retries exceeded with url: /pkgs/main/win-64/current_repodata.json (Caused by SSLError("Can't connect to HTTPS URL because the SSL module is not available."))

I do not know how to proceed. Is there a better set of instructions than what I used? Thank you.

**Comment by dalthviz** [comment]
Could you try to run from the Anaconda prompt something like the following:

```
conda create -n spyder-cf -c conda-forge spyder
```

And then:

```
conda activate spyder-cf
spyder
```

?

The error that you showed seems related with a connection error although more related with `conda` than with Spyder 🤔

**Comment by elkinm** [comment]
conda create -n spyder-cf -c conda-forge spyder fails with the same OpenSSL error as before. I could not find a solution for that. I did add conda-forge to the base environment and trying to update there, but it is taking a very long time examining conflicting packages.
Do you know when the main version in conda will update to 5.4.2 from the current 5.3.3?

**Comment by dalthviz** [comment]
I see, maybe you need to update conda to prevent the OpenSSL error? I think you can run for that something like:

    conda update -n base conda -c anaconda

After that, try again creating the new env using the conda-forge channel with the previous instructions.

Also, I would not recommended to mix channels since that could make your env unusable (so when running things on your base env always try to use the anaconda/default channel).

Regarding when Spyder latest release will be added to the anaconda channel is something we don't know, sorry.

**Comment by elkinm** [comment]
I started from the beginning, removed the environments, removed condo-forge channel. The I could reset and update base.
After that I could install simple spider environment. Then I did the final install with added packages:
`conda create -c conda-forge -n spyder-env spyder numpy scipy pandas matplotlib sympy cython`
The went in it and started spyder, no need to add the channel and it loaded 5.4.2.
And in 5.4.2 the parenthesis are not added for variables, but are added for functions as expected. Thank you.
